migrations/sql/V2021.04.06.14.53__mine_type_migration_to_permit_level.sql

```diff
@@ -0,0 +1,334 @@
+DROP INDEX IF EXISTS mine_guid_mine_tenure_type_code_permit_guid_active_uniqeness;
+
+CREATE OR REPLACE FUNCTION max (uuid, uuid)
+RETURNS uuid AS $$
+BEGIN
+	IF $1 IS NULL OR $1 < $2 THEN
+		RETURN $2;
+	END IF;
+
+	RETURN $1;
+END;
+$$ LANGUAGE plpgsql;
+
+
+CREATE AGGREGATE max (uuid)
+(
+    sfunc = max,
+    stype = uuid
+);
+
+DROP TABLE IF EXISTS multiple_permits;
+CREATE TEMP TABLE multiple_permits AS 
+	SELECT mine_guid, permit_guid FROM mine_permit_xref mpx JOIN permit p ON mpx.permit_id = p.permit_id WHERE mine_guid IN (
+	SELECT mine_guid FROM mine_permit_xref mpx JOIN permit p ON mpx.permit_id = p.permit_id
+	WHERE p.permit_status_code = 'O'
+	GROUP BY mine_guid 
+	HAVING count(p.permit_id) > 2
+);
+
+
+
+-- PLACER
+-- No related NoW
+DROP TABLE IF EXISTS mine_types;
+CREATE TEMP TABLE mine_types AS 
+SELECT m.mine_guid, p.permit_guid, mt.mine_tenure_type_code, nai.now_application_guid, na.notice_of_work_type_code, p.permit_no
+from mine_type mt
+join mine m on m.mine_guid = mt.mine_guid and mt.active_ind = true
```

Comment:
Why account for the `mt.active_ind`? Do we not also want to transfer the inactive ones?

Reply:
I guess I don't consider it a history, and don't think it has business value. I'd have to change the approach this script takes to bring inactive values across, so not sure it's worth doing.